app/controllers/users_controller.rb

```diff
@@ -1,13 +1,19 @@
 class UsersController < ApplicationController
+  before_action :correct_user, only: %i(edit update)
   before_action :find_user, only: %i(show destroy)
```

Comment:
để correct_user xuống dưới hàm find
```suggestion
  before_action :find_user, only: %i(show edit update destroy)
```